aloe_webdriver/util.py

```diff
@@ -255,6 +255,12 @@ def field_xpath(field, attribute):
         xpath = './/{field}[@{attr}=%s]'
 
     else:
+        # The string 'input-button' is used to distinguish a search
```

Comment:
Please move this to the docstring, e.g. by describing what `field` parameter is:

```
:param field: Input type to find, or 'button' to find <button> elements
```

It might be also worth it to reverse the logic and add `button-elem` to find `<button>`s, which would make both the documentation and the implementation shorter and more understandable.